src/pages/home/report/ReportTypingIndicator.js

```diff
@@ -27,7 +27,7 @@ const defaultProps = {
 };
 
 function ReportTypingIndicator(props) {
-    const usersTyping = useMemo(() => _.filter(_.keys(props.userTypingStatuses), (login) => props.userTypingStatuses[login]), [props.userTypingStatuses]);
+    const usersTyping = useMemo(() => _.filter(_.keys(props.userTypingStatuses), (accountID) => props.userTypingStatuses[accountID]), [props.userTypingStatuses]);
```

Comment:
Is `props.userTypingStatuses` at this point going to always be keyed by `accountID`?

Reply:
I like using `accountIDOrLogin` here 👍